### PR TITLE
[Cal-5] Added the calendar-modal

### PR DESCRIPTION
**Pull request by Marius1501**
Closes https://github.com/TUM-Dev/NavigaTUM/issues/289

grafik

I added the modal for the calendar when clicking on the calendar icon. I used an iframe to display the TUMOnline website but the URL is not found. I don't know what's the problem because it is the same URL which is used to open the website.

### CODE REVIEW

File: webclient/src/views/view/view-view.inc [hunk 1]
```diff
@@ -184,6 +182,33 @@
...
+          <div class="modal-header">
+            <button
+              class="btn btn-clear float-right"
+              aria-label="${{_.view_view.header.calendar.close}}$"
+              @click="calendar.modalOpen=false"
+            ></button>
+          </div>
+          <div class="modal-body">
+            <div class="content" id="modalCalendarContent">
+              <iframe src="view_data.props.calendar_url"></iframe>
```
Comment (octycs) [inline webclient/src/views/view/view-view.inc:207]:
```suggestion
              <iframe v-bind:src="view_data.props.calendar_url"></iframe>
```
I think you need to use `v-bind` here

**Comment by Marius1501** [comment]
@octycs Thank you very much for the hint! Now is the problem that the iframe hasn't the permission to show the website. 😞 
grafik

**Comment by CommanderStorm** [comment]
image
Hmm…
Yea... It looks like the easy path via an iframe is not an option here…

I have an API for the TUMonline calendar and will ask them if I can share it openly.

**Comment by CommanderStorm** [comment]
blocked until #324 is done and merged

**Comment by CommanderStorm** [comment]
@Marius1501 I am closing this issue, as I don't see a way we can get this approach working.
I think a different approach is more prommissing (see #324).

Any approaches in this way should be based on #200 though, as that will be merge-ready soon and the calendar is a thing I think would be easier to develop+test this way.

**Pull request closed without merging**